.github/workflows/onPullRequestMerged.yaml

```diff
@@ -48,9 +48,9 @@ jobs:
           commit-message: "Update v${{ steps.incremented-patch-version.outputs.version }}"
           committer: GitHub <noreply@github.com>
           author: "${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>"
-          title: 'v${{ steps.incremented-version.outputs.result }}'
+          title: 'v${{ steps.incremented-version.outputs.version }}'
           body: |
-            # v${{ steps.incremented-patch-version.outputs.version }}
+            # TBD
```

Comment:
???

Reply:
The title already have the version, no need the version in the body.